### DISCUSSION TITLE
com.rest.elevenlabs 3.2.7

### DISCUSSION THREAD

**Discussion by StephenHodgson** [RageAgainstThePixel/com.rest.elevenlabs#73] [Announcements]
## What's Changed
- enable optional settings download when requesting all voices by @IS4Code 

**Full Changelog**: https://github.com/RageAgainstThePixel/com.rest.elevenlabs/compare/3.2.6...3.2.7

<hr /><em>This discussion was created from the release <a href='https://github.com/RageAgainstThePixel/com.rest.elevenlabs/releases/tag/3.2.7'>com.rest.elevenlabs 3.2.7</a>.</em>

**Comment by stefanoditore** [comment]
Hello, I try to install your plugin in Unity two times with your pakagemanager, but I have this errroeìr:"Library\PackageCache\com.rest.elevenlabs@3.2.7\Runtime\Voices\VoicesEndpoint.cs(90,31): error CS0103: The name 'Awaiters' does not exist in the current context".

**Reply by StephenHodgson** [reply]
> Hello, I try to install your plugin in Unity two times with your pakagemanager,

It's not my package manager. It is Unity's. Did you add the scopes you needed?

- `com.utilities`
- `com.rest`

**Reply by stefanoditore** [reply]
hello, i add the scopes, but i have the same error. I try with the 3.2.6 and i not have problem. Any suggestion?

**Reply by StephenHodgson** [reply]
I'll double check and make sure 3.2.7 downloads ok
